<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>321</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_Nombre">
      <property name="text">
       <string>&amp;Nombre:</string>
      </property>
      <property name="buddy">
       <cstring>line_Nombre</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>&amp;R.U.T.</string>
      </property>
      <property name="buddy">
       <cstring>line_Rut</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_Valor">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>&amp;Valor:</string>
      </property>
      <property name="buddy">
       <cstring>line_Valor</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="line_Valor">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="line_Rut"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="line_Nombre"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="rbtn_Abono">
      <property name="text">
       <string>Abono</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtn_Deuda">
      <property name="text">
       <string>Deuda</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtn_Info">
      <property name="text">
       <string>Informaci√≥n</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pbtn_Ingreso">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingresar</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Limpiar</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plain_Info">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>line_Nombre</tabstop>
  <tabstop>line_Rut</tabstop>
  <tabstop>line_Valor</tabstop>
  <tabstop>pbtn_Ingreso</tabstop>
  <tabstop>rbtn_Abono</tabstop>
  <tabstop>rbtn_Info</tabstop>
  <tabstop>rbtn_Deuda</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>line_Nombre</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>line_Rut</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>line_Valor</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>plain_Info</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
